spec/acceptance/nodesets/el9.yml

```
@@ -0,0 +1,37 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
```

Why not shorten it to:
```suggestion
  hypervisor = ENV['BEAKER_HYPERVISOR'] || 'vagrant'
```

Or:

```suggestion
  hypervisor = ENV.fetch('BEAKER_HYPERVISOR', 'vagrant')
```